libp2p/protocols/pubsub/pubsub.nim

```diff
@@ -46,6 +46,7 @@ logScope:
 const
   KnownLibP2PTopics* {.strdefine.} = ""
   KnownLibP2PTopicsSeq* = KnownLibP2PTopics.toLowerAscii().split(",")
+  MaxMessageSize* = 1024 * 1024 # Maximum message size in bytes
```

Comment:
```suggestion
  PubsubMaxMessageSize* = 1024 * 1024 # Maximum message size in bytes
```
`MaxMessageSize` is too common, if you want to make it a global, it is not specific enough, in my opinion.

Reply:
Just pushed a commit naming it `DefaultPubsubMaxMessageSize` also to make clear that it's the default value and not a hard limit.

It got a little verbose though, let me know if `PubsubMaxMessageSize` is better and I change it